controllers/object_controls.go

```diff
@@ -2106,6 +2185,7 @@ func TransformValidatorComponent(config *gpuv1.ClusterPolicySpec, podSpec *corev
 					setContainerEnv(&(podSpec.InitContainers[i]), env.Name, env.Value)
 				}
 			}
+			setContainerEnv(&(podSpec.InitContainers[i]), DriverManagedByOperatorEnvName, fmt.Sprintf("%t", config.Driver.IsEnabled()))
```

Comment:
It's confusing that has to be set here as well as in `transformValidationInitContainer` since this seems to be looping over the init containers for the validator already.

Reply:
This has now been removed.